packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -2029,16 +2064,25 @@ describe('plugin-meetings', () => {
                   muted: false,
                   underlyingTrack: webrtcAudioTrack2
                 };
+                const fakeMicrophoneStream = {
```

Comment:
this needs to be renamed, because it overshadows `fakeMicrophoneStream` that you've already added above. The existing code in this test is calling first publishTracks with `fakeMicrophoneTrack` and then calls it again with a different track `fakeMicrophoneTrack2 `. When you change it to streams, you have to keep that logic, so you need `fakeMicrophoneStream2`

Reply:
Thanks @marcin-bazyl 
I just noticed your comment. But yes, identified and fixed this yes. Thanks again :-)